src/Dapr.Actors/Reentrancy/ActorLogicalReentrancyContext.cs

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors.Reentrancy
+{
+    using System;
+    using System.Threading;
+
+    internal static class ActorLogicalReentrancyContext
+    {
+        private static readonly AsyncLocal<string> fabActAsyncLocal = new AsyncLocal<string>();
```

Comment:
This could have a better variable name... maybe just something like `state`? I'm not sure what a `fabAct` is or why the type name is part of the variable.